zombies.js

```diff
@@ -49,14 +59,19 @@
  * @property {number} energy
  */
 
+function Food(name, energy) {
+  this.energy = energy;
+  Item.call(this, name);
+}
+Food.prototype = Object.create(Item.prototype, {
+  constructor: { value: Item }
+});
```

Comment:
This should follow structure: `DeathMetalSong.prototype = Object.create(MetalSong.prototype);`